test/libsolidity/SemanticTest.cpp

```diff
@@ -234,8 +246,23 @@ TestCase::TestResult SemanticTest::runTest(ostream& _stream, string const& _line
 			return TestResult::Failure;
 		}
 
+		if (success && !m_runWithEwasm && _compileToEwasm)
+		{
+			if (m_revertStrings != RevertStrings::Default)
+				return TestResult::Success;
```

Comment:
Why?

Reply:
There seem to be something missing in ewasm to support `RevertStrings` different from `RevertStrings::Default`. I didn't investigated this further. I just ignore these tests here. 
Maybe thats not the best solution. I will create an issue so that we will not forget this.

Reply:
Can you add a comment that explains this? It's not obvious and I wouldn't know this if I didn't find this remark in the PR.